installscripts/cookbooks/jenkins/recipes/setupjenkins.rb

```diff
@@ -7,7 +7,7 @@
   include_recipe 'cloudcli'
 
   npm_package 'serverless' do
-    version '1.26.1'
+    version '1.28.0'
```

Comment:
Any reason why you picked 1.28 instead of 1.30, since we're doing an upgrade?

Reply:
since our svcs have been tested against 1.28 i am using this.

Reply:
Since we're going to have to retest everything anyway, why not just use 1.30? If that causes problems we'll roll it up in the testing we're going to have to do anyway.

Reply:
updated